qa/integration/services/filebeat_setup.sh

```diff
@@ -8,7 +8,7 @@ if [ -n "${FILEBEAT_VERSION}" ]; then
   echo "Filebeat version is $FILEBEAT_VERSION"
   version=$FILEBEAT_VERSION
```

Comment:
Hey @jsvd do you remember if this value is set to something in the CI environment, a quick grep in the source doesn't reveal anything. I am wondering if this is set to `7.0.0-alpha1`. I am wondering in this case if we are taking the right artifact to do the test.

Reply:
this isn't being set anywhere else so the `else` clause is kicking in here.
the right way to do this should be to use the version that the test is running on. In this case, 7.0.0-alpha1-SNAPSHOT.

this means that the code must be able to detect the difference between a snapshot and a release build and fetch from the different url accordingly.

I can take ownership in creating this extra logic in a separate pr and then we can rebase this on top of master once it's merged

Reply:
Thank you @jsvd, this would be awesome.

Reply:
just wanted to say this hasn't been forgotten, still on my todo list.